tools/webtransport/h3/handler.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Tuple
+
+from .webtransport_h3_server import WebTransportSession
+
+# This file exists for documentation purpose.
```

Comment:
Indeed it's very similar to the README above. Is the README somehow extracted from this file?

Reply:
Yeah, my original plan is to generate docs from this file via Sphinx but I realized it's difficult and gave up (aioquic dependency is needed in docs/requirements.txt, some additional sphinx extensions are needed, some extensions didn't work with Sphinx 2.4.4 etc).

I want to keep this file but I'm fine with removing this file.